draft-ietf-scitt-architecture.md

```diff
@@ -1293,3 +1294,60 @@ transparent-statement:sha-256:base64url:5i6UeRzg1...qnGmr1o
 data:application/cose;base64,SGVsb...xkIQ==
 ~~~
 {: #example-transparent-statement-data-url align="left" title="Example Transparent Statement Data URL"}
+
+# Signing Statements Remotely
+
+Statements, such as digital artifacts or structured data regarding artifacts, can be too large or too sensitive to be send to a remote Transparency Services over the Internet.
+In these cases a statement can also be hash, which becomes the payload included in COSE to-be-signed bytes.
+A Signed Statement (cose-sign1) MUST be produced from the to-be-signed bytes according to {{Section 4.4 of RFC9052}}.
+
+~~~aasvg
+
+ .----+-----.
+|  Artifact  |
+ '----------'
+    |   |
+    v   |               .
+ .--+-------.          / \
+| Statement  +--+     /   \
+ '----------'   |    /     \     .----------.
+        |       +-->+  OR   +-->+  Payload   |
+        v       |    \     /     '---+------'
+ .------+---.   |     \   /          |
+|    Hash    +--+      \ /           |
+ '----+-----'           '            |
+                                     |
+
+           ...  Producer Network ...
+
+                      ...
+
+           ...   Issuer Network ...
+
+                                     |
++-----------+                        |
+| Identity  |     (iss, x5t)         |
+| Document  +--------------------+   |
++-----+-----+                    |   |
+      ^                          |   |
+      |                          |   |
+ .----+-------.                  |   |
+| Private Key  |                 |   |
+ '----+-------'                  v   |
+      |                     .----+-----.
+      |                    |   Header   |
```

Comment:
The Header part and the "To be Signed bytes are mis-leading to me"

Reply:
Specially the `Payload` sections `Arrow` enters into the Header section. What are we trying to depict?

Reply:
Also if I am not mis-taken `To be signed bytes` in our case is actually the `payload` ?

If it is, then can we keep the two same so as to logically link the two sections of the diagrams without physically linking them?